pint/util.py

```diff
@@ -615,6 +615,10 @@ def _check(self, other):
         same unit registry.
 
         """
+        if type(other).__name__ == 'PintArray':
+            print("notimpl")
```

Comment:
I guess this should be removed for the final version :-)

Reply:
And also, I do not understand the purpose of this. Why not directly adding the SharedRegistryObject to all pandas-pint related objects. Is this because everything is stored as strings in a dataframe?

Reply:
no idea... @andrewgsavage ?

Reply:
This can be deleted, it was added when I was trying to get quantity + pintarray (or any operation) to return NotImplemented